.jenkins/pytorch/test.sh

```diff
@@ -455,32 +455,49 @@ test_xla() {
 }
 
 # Do NOT run this test before any other tests, like test_python_shard, etc.
-# Because this function uninstalls the torch built from branch, and install
-# nightly version.
+# Because this function uninstalls the torch built from branch and installs
+# the torch built on its base commit.
 test_forward_backward_compatibility() {
   set -x
+  REPO_DIR=$(pwd)
+  if [[ "${BASE_SHA}" = "${SHA1}" ]]; then
```

Comment:
@seemethere is the master of bash, but I think we are using double equals throughout the codebase (although single equal is perfectly fine, but most of us have C++/Python background, where `=` is an assignment operator and `==` is an equality one
```suggestion
  if [[ "${BASE_SHA}" == "${SHA1}" ]]; then
```